src/components/SingleName/NameDetails.js

```diff
@@ -520,8 +520,7 @@ function NameDetails({
   tab,
   pathname
 }) {
-  // const duringMigration = process.env.REACT_APP_MIGRATION_COMPLETE !== 'True'
-  const duringMigration = true
+  const duringMigration = process.env.REACT_APP_MIGRATION_COMPLETE !== 'True'
```

Comment:
`REACT_APP_MIGRATION_COMPLETE` is taken out.

Reply:
aprove

Reply:
???